packages/perseus/src/widgets/interactive-graphs/graphs/point.tsx

```diff
@@ -80,13 +81,19 @@ export function UnlimitedPointGraph(props: PointGraphProps) {
                     onMove={(destination) =>
                         dispatch(actions.pointGraph.movePoint(i, destination))
                     }
-                    onFocusChange={(isFocused) => {
+                    onFocusChange={(event, isFocused) => {
                         if (isFocused) {
                             dispatch(actions.pointGraph.focusPoint(i));
                         } else {
+                            if (event.relatedTarget?.id === REMOVE_BUTTON_ID) {
```

Comment:
This prevents the button from stealing focus and lets us remove the `previouslySelectedPoint` field on the reducer. This makes this MUCH more tidy and straightforward to follow the logic. The downside is that this component needs to know about another components existence. To prevent these components from slipping away from each other I use a constant `REMOVE_BUTTON_ID` to make sure that should the id of that button ever change that this will keep working

Reply:
TIL about `relatedTarget`!